charts/sigstore-prober/templates/deployment.yaml

```diff
@@ -30,6 +30,10 @@ spec:
         volumeMounts:
         - name: sigstore
           mountPath: "/var/run/sigstore"
+        securityContext:
+          runAsNonRoot: true
+          runAsUser: 65533
```

Comment:
where does this ID come from?

Reply:
same question :) i checked the image and that does not set a user/group

Reply:
just a typo, good catch